ntp.sls

```diff
@@ -9,7 +9,7 @@
     {% set PROGRAM_FILES = "%ProgramFiles%" %}
 {% endif %}
 ntp:
-  {% for version in ['4.2.8p11'] %}
+  {% for version in ['4.2.8p11', '4.2.8p12', '4.2.8p13'] %}
```

Comment:
Can you reverse the order here... newer versions first. Makes it easier to add when you have 100 versions.

Reply:
Sure, done now!